package/MDAnalysis/coordinates/base.py

```diff
@@ -141,9 +141,15 @@
 from ..auxiliary.core import get_auxreader_for
 
 class Namespace(object):
-    """Empty class 
-    To allow storing attributes in new namespace """
-    pass
+    """Class to allow storing attributes in new namespace. """
```

Comment:
You could have a look how pandas allows both at the same time for it's dataframe objects.